docs/configurations.md

```diff
@@ -0,0 +1,235 @@
+| `scalar.db.consensus_commit.parallel_commit.enabled`      | Whether or not the commit phase is executed in parallel.                       | `true`                                                            |
+| `scalar.db.consensus_commit.parallel_rollback.enabled`    | Whether or not the rollback phase is executed in parallel.                     | The value of `scalar.db.consensus_commit.parallel_commit.enabled` |
+| `scalar.db.consensus_commit.async_commit.enabled`         | Whether or not the commit phase is executed asynchronously.                    | `false`                                                           |
+| `scalar.db.consensus_commit.async_rollback.enabled`       | Whether or not the rollback phase is executed asynchronously.                  | The value of `scalar.db.consensus_commit.async_commit.enabled`    |
+
+#### Underlying storage or database configurations
+
+Consensus Commit has a storage abstraction layer and supports multiple underlying storages. You can specify the storage implementation by using the `scalar.db.storage` property.
+
+Select a database to see the configurations available for each storage.
+
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Cassandra', 'tabset-1')" id="defaultOpen-1">Cassandra</button>
+  <button class="tablinks" onclick="openTab(event, 'CosmosDB_for_NoSQL', 'tabset-1')">CosmosDB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB', 'tabset-1')">DynamoDB</button>
+  <button class="tablinks" onclick="openTab(event, 'JDBC_databases', 'tabset-1')">JDBC databases</button>
+</div>
+
+<div id="Cassandra" class="tabcontent" markdown="1">
+
+The following configurations are available for Cassandra:
+
+| Name                                    | Description                                                           | Default    |
+|-----------------------------------------|-----------------------------------------------------------------------|------------|
+| `scalar.db.storage`                     | `cassandra` must be specified.                                        | -          |
+| `scalar.db.contact_points`              | Comma-separated contact points.                                       |            |
+| `scalar.db.contact_port`                | Port number for all the contact points.                               |            |
+| `scalar.db.username`                    | Username to access the database.                                      |            |
+| `scalar.db.password`                    | Password to access the database.                                      |            |
+| `scalar.db.cassandra.metadata.keyspace` | Keyspace name for the namespace and table metadata used for ScalarDB. | `scalardb` |
```

Comment:
Since this PR is for `3.8` and earlier, I don't think this configuration should be in this doc. Is that correct?

Reply:
Yes exactly.